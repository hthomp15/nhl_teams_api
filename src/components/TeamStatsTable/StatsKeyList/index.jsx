import React from "react";

function StatsKeyList() {
    return (
        <table className="table-auto w-1/3 text-center border-2">
            <tbody>
                <tr className="border border-white">
                    <th className="border border-white p-1">GP</th>
                    <td>Games Played</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">W</th>
                    <td>Wins</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">L</th>
                    <td>Losses</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">PTS</th>
                    <td>Points</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">OT</th>
                    <td>Overtime Games</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">GPG</th>
                    <td>Goals Per Game</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">GAPG</th>
                    <td>Goals Against Per Game</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">SV%</th>
                    <td>Save Percentage</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">S</th>
                    <td>Shots</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">SA</th>
                    <td>Shots Allowed</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">S%</th>
                    <td>Shooting Percentage</td>
                </tr>
                <tr className="border border-white">
                    <th className="border border-white p-1">FOW%</th>
                    <td>Faceoff Win Percentage</td>
                </tr>
            </tbody>
        </table>
    );
}

export default StatsKeyList;