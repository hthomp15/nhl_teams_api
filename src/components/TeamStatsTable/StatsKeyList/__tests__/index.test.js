import React from "react";
import { render, cleanup } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';

import StatsKeyList from "..";

afterEach(cleanup);

describe("StatsKeyList Component", () => {
    it("renders", () => {
        render(<StatsKeyList />);
    });
    it("matches snapshot DOM node structure", () => {
        const { asFragment } = render(<StatsKeyList />);
        expect(asFragment()).toMatchSnapshot();
    })
});