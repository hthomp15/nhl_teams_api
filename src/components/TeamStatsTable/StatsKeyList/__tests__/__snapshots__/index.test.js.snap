// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsKeyList Component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <table
    class="table-auto w-1/3 text-center border-2"
  >
    <tbody>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          GP
        </th>
        <td>
          Games Played
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          W
        </th>
        <td>
          Wins
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          L
        </th>
        <td>
          Losses
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          PTS
        </th>
        <td>
          Points
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          OT
        </th>
        <td>
          Overtime Games
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          GPG
        </th>
        <td>
          Goals Per Game
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          GAPG
        </th>
        <td>
          Goals Against Per Game
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          SV%
        </th>
        <td>
          Save Percentage
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          S
        </th>
        <td>
          Shots
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          SA
        </th>
        <td>
          Shots Allowed
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          S%
        </th>
        <td>
          Shooting Percentage
        </td>
      </tr>
      <tr
        class="border border-white"
      >
        <th
          class="border border-white p-1"
        >
          FOW%
        </th>
        <td>
          Faceoff Win Percentage
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
